### DISCUSSION TITLE
Direction finding code support for nrf52811?

### DISCUSSION THREAD

**Discussion by Bosemani** [zephyrproject-rtos/zephyr#43551] [General]
Hai
I'm trying to implement Direction-finding in the nrf52811 chip.
i found [this code ](https://github.com/zephyrproject-rtos/zephyr/tree/main/samples/bluetooth/direction_finding_connectionless_tx)
I want to use this code for the nrf52811 chip.
But in that code, they didn't mention it will support nrf52811.
My question is
Is that code support the nrf52811 chip?
if it supports what are the changes need to do?
can you explain it,

Thank you..

Regards 
Bose

**Comment by carlescufi** [comment]
@ppryga ^^

**Comment by ppryga-nordic** [comment]
Hello @Bosemani 

That is easy. Copy contents of these files: 
https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52833.conf 
to `samples/bluetooth/direction_finding_connectionless_tx/boards/boards/nrf52840dk_nrf52811.conf`,
https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52833.overlay
to `samples/bluetooth/direction_finding_connectionless_tx/boards/boards/nrf52840dk_nrf52811.overlay`.

Update https://github.com/zephyrproject-rtos/zephyr/blob/681127f6ce019318c0eddfcad420081bf5e839b2/samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52833.overlay#L22-L25 with GPIO pins you connect to antenna matrix if you want to use the sample in AoD mode.

I don't have the hardware to check it in runtime, so in case you have some issue just write in this thread.
